spec/requests/claims_spec.rb

```diff
@@ -178,6 +181,10 @@
             expect(in_progress_claim.submitted_at).to be_present
           end
 
+          it "sends an email" do
+            expect(claim_mailer_spy).to have_received(:deliver_later)
+          end
+
```

Comment:
Think we could also test the negative case: we don't send an email if submission fails.